docs/lesson-plans/week-1.md

```diff
@@ -0,0 +1,726 @@
+
+`ls` is also a very commonly used commands. It lists all of the contents of a directory (both files and 
+sub-directories).
+
+Format:
+```
+ls <optional filepath>
+```
+
+- You can list hidden files in a directory with the `-a` flag
+
+Examples:
+```
+# List files in the current directory
+ls
+
+# List all files in the current directory
+ls -a
+
+# List files in the Desktop directory
+ls Desktop
+```
+
+#### pwd
+
+`pwd` stands for 'print working directory', and it logs the current directory you're located at.
+
+Format:
+```
+pwd
+```
+
+#### mv
+
+`mv` is a command that lets you move a file or directory to another location. It can also be used
+to rename a file or directory (since that's the same thing as moving it in a way).
+
+Format:
+```
+mv <from filepath> <to filepath>
+```
+
+- You can't move a file inside of itself or one of its subdirectories (without messing 
+around with symbolic links, but that's way outside of the material of this course).
+
+Example:
+```
+# Move the README.md file to the parent directory without renaming it
+mv README.md ../
+
+# Rename the test.txt file to test.md
+mv test.txt test.md
+
+# Move the test.txt file to the Desktop from Documents and rename it to hello.world
+mv Documents/test.txt Desktop/hello.world
+```
+
+#### cp
+
+`cp` copies a file, and it's very similar to `mv`. 
+
+Format:
+```
+cp <exiting file or directory> <new file or directory>
+```
+
+- If you're copying a directory, you need to make sure to include the `-r` flag so that it 
+copies everything in the directory too.
+
+Examples:
+```
+# Copy test.txt to Desktop
+cp test.txt Desktop
+
+# Copy the Desktop directory to Documents 
+cp Desktop Documents -r
+```
+
+#### mkdir
+
+`mkdir` creates a directory.
+
+Format:
+```
+mkdir <directory name>
+```
+
+Examples:
+```
+# Create a Desktop2 directory
+mkdir Desktop2
+```
+
+#### rm
+
+`rm` deletes a file or a directory.
+
+```
+rm <path to file or directory>
+```
+
+- Be careful using this command. Once it's used, files are pretty much unrecoverable without special software.
+- If you're deleting a directory, you'll need to pass in the `-rf` flags.
+    - The `-r` flag makes it recursive, it will delete all subdirectories.
+    - The `-f` flag is called force, and it's sometimes required to delete a non-empty directory.
+- Never try to delete the `/` directory using `rm -rf /`. It is a very dangerous command that will
+end up deleting your entire computer (there are a couple protections, but would you really want
+to test them out?).
+
+```
+# Delete the test.txt file
+rm test.txt
+
+# Delete the test directory
+rm -rf test
+```
+
+#### cat
+
+`cat` prints out the contents of a file. It technically stands for 'concatenate' file to 
+standard output (your console).
+
+Format:
+```
+cat <filepath>
+```
+
+```
+# Print out contents of test.txt
+cat test.txt
+```
+
+#### vim/vi
+
+`vim` (sometimes also `vi`) is a text editor that usually comes preinstalled on most terminals. It allows you
```

Comment:
`vi` is technically a different text editor than `vim`. Maybe just make this section about vim and have like a footnote or something for `vi`

Reply:
In a block quote now